app/assets/stylesheets/navbar.scss

```diff
@@ -0,0 +1,88 @@
+#navbar-nav ul li {
+  text-align: center;
+  text-transform: uppercase;
+}
+
+.home-page-text {
+  color: $white;
+  font-weight: 500;
+}
+
+#logo-text {
+  padding: 4px 5px;
+  border: 0px;
+  line-height: 12px;
+  height: 4px;
+  font-size: 11px;
+  font-weight: 500;
+  letter-spacing: 1px;
+}
+
+.logo-img {
+  width: 30px;
+  height: 30px;
+  border-radius: 15px;
+}
+
+#menu-top li {
+  line-height: 8px;
+  font-size: 11px;
+  letter-spacing: 2px;
+  transition: transform .2s;
+}
+
+.title {
+  font-weight: 525;
+  line-height: 20px;
+  border: 0;
+}
+
+#menu-top li:hover {
+  transform: scale(1.2);
+}
+
+.navbar-toggler {
+  transition: transform .2s;
+}
+
+.navbar-toggler:hover {
+  transform: scale(1.1);
+}
+
+.fa-utensils {
+  color: $white;
+}
+
+@media (max-width: 992px) {
```

Comment:
Use [bootstrap's built in mixins for media queries](https://getbootstrap.com/docs/4.0/layout/overview/#responsive-breakpoints)

Reply:
solved. ✓